lib/ocfl/layouts/druid_tree.rb

```diff
@@ -16,6 +31,37 @@ def self.path_to(identifier)
           File.join(segments)
         )
       end
+
+      private
+
+      def extension_name
+        "0010-differential-n-tuple-omit-prefix-storage-layout"
+      end
+
+      def extension_directory
+        base_directory / EXTENSIONS_DIRECTORY / extension_name
+      end
+
+      def to_config_json
+        JSON.generate(
+          {
+            extensionName: "0010-differential-n-tuple-omit-prefix-storage-layout",
```

Comment:
Should this use the `extension_name` method as it is on line 59?

Reply:
💯, thanks!